l10n_br_fiscal/models/document_fiscal_mixin_methods.py

```diff
@@ -53,6 +53,10 @@ def _compute_amount(self):
                             values[field] += line[field.replace("amount_", "")]
             doc.update(values)
 
+    def _compute_country(self):
+        for record in self:
+            record.country_id = self.env.company.country_id.id
+
```

Comment:
I don't think it's a good option to take the currently active company, the company that owns the document could be another.